### ISSUE TITLE
Make View PDF a toggle (or add a toggle)

### ISSUE THREAD

**Issue by memeplex** [James-Yu/LaTeX-Workshop#1391]
**Describe the solution you'd like**

The PDF view is annoying when you're not in the previewed tex file and also a reasonable workflow could be to quickly open and close it. Currently there are some shortcomings:

1. No provided key to quickly hide it.
2. The preview is unresponsive to most standard shortcuts, so even the cumbersome: go there and close it is inviable.

The Markdown All in One extension makes Control-Shift-V and Control-K V toggle the preview. I think this is a good and simple solution. In this case:

1. If there is an open and visible preview for the current project, close it (or hide it, if that's possible in vscode).
2. If there is an open but non-visible preview for the current project, make it visible (but don't necessarily focus it).
3. In any other case, open a preview.

This behavior could be made the default, could be made into a new command or could be enabled by a settings toggle.

**Comment by jlelong** [comment]
When the PDF viewer has focus, all keystrokes are intercepted by `pdf.js`, which is a big difference with what happens in the Markdown All in One extension. I do not know if this is possible to call a VSCode command from inside `pdf.js`.

**Comment by memeplex** [comment]
The toggle would be very helpful even if not available in the preview itself.

Not having some "window anagement" keys working in the preview tab is certainly annoying but we could open another issue for that.

**Comment by memeplex** [comment]
Also, it would be nice if the "View in pdf" links in the sections hover popups did the same than I described above (except for closing the preview). Currently they don't even open the preview, leaving the user with the impression that something is not working fine.

**Comment by jlelong** [comment]
The `View in pdf` links call `synctex` so they require a `PDF` viewer to be opened. I have just added more details on that to the [wiki](https://github.com/James-Yu/LaTeX-Workshop/wiki/Hover#previewing-references).

Accessing the list of opened editors from VSCode's API is not possible, see [vscode#15178](https://github.com/microsoft/vscode/issues/15178) and therefore closing editors from the extension side is even trickier, see [vscode#21617](https://github.com/microsoft/vscode/issues/21617). These issues do not seem to progress much. I do not think we can do much to improve the current behaviour of the `PDF` viewer tab without using horrible hacks. 

The Markdown All in One's toggle preview function only closes the preview tab when it has the focus. To be more precise, when the focus is not in a `textEditor`, the toggle preview function closes the preview.

**Comment by memeplex** [comment]
Ok, so short of tracking open/close events, you are unable to open a preview if there isn't one already, let alone closing it which is not even possible. It surprises me vscode API shows such a limitation, I mean, I came from emacs and from day one I realized it has a quite restrictive extension model (maybe for good), but this seems too much.

Maybe "View in pdf" should be renamed to "Sync preview" or "Sync pdf to this position".
